src/screens/SplashScreen/index.js

```diff
@@ -21,7 +22,7 @@ const SplashScreen = () => {
     navigation.dispatch(StackActions.replace(selfUserId ? 'HomeTabs' : 'SignIn'));
   };
 
-  const {setProfileId} = useProfileHook();
+  const {setAuth} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `setAuth` from `useUserAuthHook` is correct as it replaces the previous `setProfileId` from `useProfileHook`.
```diff
- const {setProfileId} = useProfileHook();
+ const {setAuth} = useUserAuthHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->